__tests__/index.test.js

```diff
@@ -36,4 +36,20 @@ describe('GET /api/topics', () => {
       })
     })
   })
+})
+
+describe('GET /api', () => {
```

Comment:
If this is an attempt to test the endpoints.json just once and never again, that is quite lazy. Jest matchers to check object contents will allow you to check that the file contains the latest changes you've made in a non brittle way as you update the file.